app/assets/javascripts/map/presenters/analysis/AdvancedAnalysisResultsPresenter.js

```diff
@@ -0,0 +1,41 @@
+define([
+  'map/presenters/PresenterClass',
+  'underscore',
+  'backbone',
+  'mps',
+], function(PresenterClass, _, Backbone, mps) {
+
+  'use strict';
+
+  var AdvancedAnalysisResultsPresenter = PresenterClass.extend({
+
+    init: function(view) {
+      this.view = view;
+      this._super();
+    },
+
+    _subscriptions: [{
+      'AnalysisService/results': function(results) {
+        if (results.failure !== undefined) {
+          return this.view._renderFailure();
+        }
+
+        if (results.meta.id === 'loss-by-type') {
```

Comment:
I'm getting an error here: "Uncaught TypeError: Cannot read property 'id' of undefined"